man/docker-daemon.8.md

```diff
@@ -464,6 +465,16 @@ this topic, see
 Otherwise, set this flag for migrating existing Docker daemons to a
 daemon with a supported environment.
 
+Here is supported options of *zfs*:
```

Comment:
I think we should make this a heading, for example:

```Markdown
## ZFS options
```

And do the same for [line 262](https://github.com/zhuguihua/docker/blame/fix_storage_driver_options_in_man_page/man/docker-daemon.8.md#L262); change "Here is the list of *devicemapper* options:" to

```Markdown
## Devicemapper options
```